samples/bluetooth/peripheral_uart/README.rst

```diff
@@ -93,20 +109,23 @@ Testing
 
 After programming the sample to your development kit, test it by performing the following steps:
 
-1. Connect the kit to the computer using a USB cable. The kit is assigned a COM port (Windows) or ttyACM device (Linux), which is visible in the Device Manager.
+1. Connect the device to the computer to access UART 0.
+   If you use a development kit, UART 0 is forwarded as a COM port (Windows) or ttyACM device (Linux) after you connect the development kit over USB.
+   If you use Thingy:53, you must attach the debug board and connect an external USB to UART converter to it.
 #. |connect_terminal|
-#. Optionally, connect the RTT console to display debug messages. See :ref:`peripheral_uart_debug`.
+#. Optionally, you can display debug messages. See :ref:`peripheral_uart_debug` for details.
 #. Reset the kit.
 #. Observe that **LED 1** is blinking and that the device is advertising with the device name that is configured in :option:`CONFIG_BT_DEVICE_NAME`.
 #. Observe that the text "Starting Nordic UART service example" is printed on the COM listener running on the computer.
 #. Connect to the device using nRF Connect for Mobile.
    Observe that **LED 2** is on.
-#. Optionally, pair or bond with the device with MITM protection. This requires :ref:`RTT connection <testing_rtt_connect>`.
+#. Optionally, pair or bond with the device with MITM protection.
+   This requires using the passkey value displayed in debug messages.
    To confirm pairing or bonding, press **Button 1** on the device and accept the passkey value on the smartphone.
 #. In the application, observe that the services are shown in the connected device.
 #. Select the UART RX characteristic value in nRF Connect.
    You can write to the UART RX and get the text displayed on the COM listener.
-#. Type '0123456789' and tap :guilabel:`Write`.
+#. Type '0123456789' and tap :guilabel:`Send`.
```

Comment:
Why "Send" instead of "Write" ?

Reply:
Testing instructions refer to `nRF Connect for Mobile`. When I used this application, the button was labeled send instead of write.

Reply:
Fixed capitalization of `SEND` to be more consistent with Android application